dallinger/db.py

```diff
@@ -53,6 +53,10 @@
 """
 
 
+def create_db_engine(db_url, pool_size=1000):
```

Comment:
This seems like a good idea, but there's still the definition of `engine` above. I guess we need to retain that for backwards compatibility, but maybe it should be defined using this function instead?

Reply:
I initially thought about a switch to creating the `engine` global with the new function, but I felt like the import-time creation of `engine` in this module was OK as-is because it's "private" to the module, and the fact that it's the result of a sqlalchemy call is hidden from client code referencing `db.engine` from outside. I introduced `create_db_engine()` in the spirit of "just give me whatever db engine we use in Dallinger, but for _this DB URL_ instead of what's in the environment var, and I don't generally care about things like pool_size".